docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+
+  web:
+    build:
+      context: .
+      args:
+        - PORT=8000
+        - VERSION=${VERSION:-latest}
+    ports:
+      - "${WEB_PORT:-8000}:8000"
```

Comment:
What is this variable for?

Reply:
there is no variable, so I changed the syntax